RecoBTag/SecondaryVertex/interface/V0Filter.h

```diff
@@ -6,6 +6,7 @@
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 
 #include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
```

Comment:
@mrodozov : since you are there, there are a few includes that can be removed from the .cc implementation file, then...